config.yaml

```diff
@@ -26,6 +26,41 @@ log:
     ## to set whether or not the SDK key is included in the logging output.
     includeSdkKey: true
 
+##
+## tracing: tracing configuration
+##
+tracing:
```

Comment:
What about "opentelemetry"? I do not strong opinion on this, but this tracing config defintion is for OTEL only and the name will make it clear.
Or we can also consider "tracing > opentelemetry" for extensibility.

Reply:
Yeah, love the idea. But OpenTelemetry actually provides all tracing, metrics, logging. We can also think about "opentelemetry" > "tracing". 

About extensibility, I think the currect model is extensible in terms of the trace data that can be exported anywhere. But we want to collect trace data in many ways then extensibility comes into the picture. But collecting the same trace data in many ways might not be that important.

would love discuss more on this